package.json

```diff
@@ -2,10 +2,40 @@
   "name": "warehouse.ai",
   "version": "7.0.0",
   "description": "Scalable Object Ledger and CDN",
-  "scripts": {},
+  "main": "lib/app.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "dependencies": {
+    "fastify": "^3.0.0",
+    "fastify-auth": "^1.0.1",
+    "fastify-autoload": "^3.3.0",
+    "fastify-plugin": "^2.3.0",
+    "pino": "^6.7.0"
+  },
+  "devDependencies": {
+    "eslint": "^7.10.0",
+    "eslint-config-godaddy": "^4.0.0",
+    "eslint-plugin-json": "^2.1.2",
+    "eslint-plugin-mocha": "^8.0.0",
+    "fastify-cli": "^2.3.0",
+    "nodemon": "^2.0.4",
+    "standard-version": "^9.0.0",
+    "tap": "^14.0.0"
```

Comment:
Why not `mocha`?

Reply:
`tap` is the preferred testing framework for `fastify` applications: https://www.fastify.io/docs/latest/Testing/

However, I haven't really thought about testing yet. I left this devDep from the default projects that gets generated with `npm init fastify`.

Reply:
Before writing the first test we can look into `mocha` vs `tap` deeper and choose the one we prefer.